package exceptiondemo;

public class Demo {
	public static void main(String []args) {	try {	int []numbers= {1,2,3};
		System.out.println(numbers[10]);}
	catch (Exception e) {
		// TODO: handle exception
		System.out.println("vanakam");
		e.printStackTrace();} 
	//int a= 10/0	;
	//System.out.println("vanakam chennai");
		
		
	}
		
	
	


}
